src/api_manager/auth/lib/auth_jwt_validator.cc

```diff
@@ -304,12 +304,12 @@ void JwtValidatorImpl::UpdateAudience(grpc_json *json) {
   }
 }
 
-grpc_jwt_verifier_status JwtValidatorImpl::ParseImpl() {
+Status JwtValidatorImpl::ParseImpl() {
   // ====================
   // Basic check.
   // ====================
   if (jwt == nullptr || jwt_len <= 0) {
-    return GRPC_JWT_VERIFIER_BAD_FORMAT;
+    return ToStatus("Bad JWT format: empty token");
```

Comment:
what is the error code by using this ToStatus?

Reply:
It is Code::UNAUTHENTICATED, please see line 173